spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'voxpupuli/acceptance/spec_helper_acceptance'
+
+ENV['BEAKER_FACTER_ICINGA_REPO_USER'] = ENV.fetch('ICINGA_REPO_USER', nil)
+ENV['BEAKER_FACTER_ICINGA_REPO_PASSWORD'] = ENV.fetch('ICINGA_REPO_PASSWORD', nil)
```

Comment:
Can we standardize on the `BEAKER_FACTER_*` env vars and leave this out?

Reply:
Of course, but what exactly do you mean by that?